Earthfile

```diff
@@ -3,16 +3,17 @@ VERSION 0.7
 FROM busybox:1.32.0
 
 hello:
-    COPY globe.txt ./
-    RUN cat globe.txt
-
-hello-name:
     ARG name=world
-    RUN echo "Hello, $name!"
+    COPY globe.txt /globe.txt
+    RUN cat /globe.txt
+    RUN echo "Hello, $name!" > /hello.txt
+    CMD ["cat", "/hello.txt"]
+    SAVE ARTIFACT /hello.txt AS LOCAL hello.txt
```

Comment:
Probably a longshot but possibly a user might lose a file they did not expect to lose when running this?

Reply:
True... any ideas?

Reply:
I think anything with earthly in it should be fine (but if we want to be extra safe we can use a pregenerated random string (so that cache won't bust) - 

e.g `hello-from-earthly-technologies.txt`, `hello-earthly-349875894357.txt`

Reply:
I'll merge as is for now, and I might fix this later.